src/microbe_stage/OrganelleLayout.cs

```diff
@@ -72,25 +72,30 @@ public bool CanPlace(T organelle, bool allowCytoplasmOverlap = false)
 
     /// <summary>
     ///   Returns true if CanPlace would return true and an existing
-    ///   hex touches one of the new hexes.
+    ///   hex touches one of the new hexes, or is the last hex and replaced.
     /// </summary>
-    public bool CanPlaceAndIsTouching(T organelle, bool allowCytoplasmOverlap = false)
+    public bool CanPlaceAndIsTouchingOrReplacingLast(T organelle, bool allowCytoplasmOverlap = false)
     {
         if (!CanPlace(organelle, allowCytoplasmOverlap))
             return false;
 
-        return IsTouchingExistingHex(organelle);
+        return IsTouchingExistingHexOrReplacingLast(organelle);
```

Comment:
`IsTouchingExistingHex(organelle) || (allowReplacingLastCytoplasm && IsReplacingLast(...));`

Reply:
Changed in  f3ad887, seems to work.